src/test/kotlin/uk/gov/dluhc/printapi/mapper/VacSearchSummaryResponseMapperTest.kt

```diff
@@ -0,0 +1,122 @@
+package uk.gov.dluhc.printapi.mapper
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.mockito.InjectMocks
+import org.mockito.Mock
+import org.mockito.junit.jupiter.MockitoExtension
+import org.mockito.kotlin.any
+import org.mockito.kotlin.given
+import uk.gov.dluhc.printapi.dto.PrintRequestStatusDto
+import uk.gov.dluhc.printapi.models.PrintRequestStatus
+import uk.gov.dluhc.printapi.models.VacPrintRequestSummary
+import uk.gov.dluhc.printapi.models.VacSearchSummaryResponse
+import uk.gov.dluhc.printapi.models.VacSummaryResponse
+import uk.gov.dluhc.printapi.testsupport.testdata.dto.buildVacPrintRequestSummaryDto
+import uk.gov.dluhc.printapi.testsupport.testdata.dto.buildVacSearchSummaryResults
+import uk.gov.dluhc.printapi.testsupport.testdata.dto.buildVacSummaryDto
+import java.time.OffsetDateTime
+
+@ExtendWith(MockitoExtension::class)
+class VacSearchSummaryResponseMapperTest {
+
+    @Mock
+    private lateinit var instantMapper: InstantMapper
+
+    @Mock
+    private lateinit var printRequestStatusMapper: PrintRequestStatusMapper
+
+    @InjectMocks
+    private lateinit var mapper: VacSummarySearchResponseMapperImpl
+
+    @Test
+    fun `should map VacSummaryDto to an VacSummaryResponse`() {
```

Comment:
I feel horrible only writing nitpick comments :/ But "an" --> "a" here and below